ui-v2/app/templates/dc/services/show/upstreams.hbs

```diff
@@ -0,0 +1,32 @@
+<div id="upstreams" class="tab-section">
+  <div role="tabpanel">
+  {{#if (gt gateway.Services.length 0)}}
+    <section>
+      <p>
+        Upstreams are services that may recieve traffic from this gateway. Learn more about configuring gateways in our
+        <a href={{env 'CONSUL_INGRESS_GATEWAYS_URL'}} target="_blank">documentation.</a>
+      </p>
+      <ListCollection @cellHeight={{73}} @items={{gateway.Services}} class="consul-gateway-services-list" as |item index|>
+        <a data-test-service-name href={{href-to 'dc.services.show' item.Name}}>
+          {{item.Name}}
+        </a>
+        <ul>
+        {{#if (not-eq item.GatewayConfig.ListenerPort 0)}}
+          <li class="port">
+            <CopyButton
+              @value={{item.GatewayConfig.ListenerPort}}
+              @name="Address"
+            />
+            <span>:{{item.GatewayConfig.ListenerPort}}</span>
```

Comment:
I know the gateway config only includes a port, but is there something we can infer here. Like I wonder if the host address is the address of something else? Like is it the address of the gateway service?

Reply:
I'll bring this up to the designers.

Reply:
Changed to `Port`